benchmarks/timm_models_list.txt

```diff
@@ -0,0 +1,69 @@
+adv_inception_v3 128
```

Comment:
@ngimel These are the batch sizes. This PR also contains a batch size finder, which finds the largest eager batch size that could fit on the GPU. These batch sizes are computed using that finder,